src/Elastic.Apm/Config/ConfigConsts.cs

```diff
@@ -1,12 +1,22 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace Elastic.Apm.Config
 {
 	public static class ConfigConsts
 	{
 		public static class Constraints
 		{
+			public const string LogLevel = "ELASTIC_APM_LOG_LEVEL";
```

Comment:
I think none of the new consts added here from `Constraints` is used. These new values seem to be the same as values from `EnvVarNames` - could we maybe remove these?

Reply:
Done.